hhvm/README-short.txt

```diff
@@ -0,0 +1 @@
+HHVM is an open PHP engine build by Facebook.
```

Comment:
build => built 

but maybe just use their own wording:
"HHVM is an open-source virtual machine designed for executing programs written in Hack and PHP" + ", made by Facebook."